### PR TITLE
SEN-1453 -- create repo and first commit for @cascadeenergy/config-client

### PR DESCRIPTION
**Pull request by notxt**
**Testing:**
- run gulp tests
- download testConfigClient.zip from ticket
- upload to ec2
- upload branch code to `testConfigClient/@cascadeenergy/config-client`
- run `CONSUL_PORT_8500_TCP_ADDR='1[ Consul IP ]' node testConfigClient.js --key [ path/to/consul/value ]`

**Ticket:**
https://cascadeenergy.atlassian.net/browse/SEN-1453

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "gulp-plumber": "^1.0.0",
+    "isparta": "^3.0.3",
+    "jshint-stylish": "^1.0.2",
+    "mocha": "^2.2.4",
+    "nock": "^2.6.0",
+    "require-dir": "^0.3.0"
+  },
+  "dependencies": {
+    "babel-runtime": "^5.2.17",
+    "bluebird": "^2.9.30",
```
Comment (notxt) [inline package.json]: used to

- [ ] remove bluebird

File: es6/test/index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import configFactory from '../index';
+
```
Comment (notxt) [inline es6/test/index.js]: - [ ] remove extra line

**Comment by notxt** [comment]
if consul-client should be a public module, then probably should be one as well, how do we feel about the name `configerator`

**Comment by nackjicholson** [comment]
Yup, make it public as well I think. I'm fine with your name. Maybe with a "u" though...? `configurator`, if it's available.

For parody with the other one, consider `consul-kv` or `consul-config`

**Comment by notxt** [comment]
- [ ] rename repo to `consul-kv`

File: es6/test/index.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import assert from 'assert';
+import nock from 'nock';
+
+
+describe('Config Client', () => {
+  const host = 'key.value.service';
+  const url = 'http://' + host + '/v1/kv/';
```
Comment (notxt) [inline es6/test/index.js]: - [ ] move vars down

**Pull request closed without merging**